src/link/assign.c

```diff
@@ -42,7 +67,7 @@ area_Avail(SLONG bank)
 	return (r);
 }
 
-SLONG 
+bool
```

Comment:
I'd stick to `int` in case we need to add error codes in the future... I don't like to use `bool` in any kind of complicated code.